docs/blog/2021-07-17-zephyr-2-5.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Zephyr 2.5 Update"
+author: Pete Johanson
+author_title: Project Creator
+author_url: https://gitlab.com/petejohanson
+author_image_url: https://www.gravatar.com/avatar/2001ceff7e9dc753cf96fcb2e6f41110
+tags: [firmware, zephyr, core]
+---
+
+I'm happy to announce that we have completed the [work](https://github.com/zmkfirmware/zmk/pull/736/) to upgrade ZMK to [Zephyr 2.5](https://docs.zephyrproject.org/2.5.0/releases/release-notes-2.5.html)!
+
+A big part of this work was some _major_ refactors and improvements by [innovaker] to our [zmk-docker](https://github.com/zmkfirmware/zmk-docker/) Docker image and GH Actions automation.
+
+- Faster build times with improved caching.
+- Integration tests which automatically verify new images.
+- PRs to the repo now build properly and run the tests as well.
+- Build images for multiple target architectures, e.g. `zmk-build-riscv64`, all in parallel.
+- Nightly builds to be sure we're pulling in the latest OS/package updates, to ensure we keep our images up to date, address any reported vulnerabilities, etc.
+- Faster upgrade paths for future Zephyr SDK and Zephyr versions.
+
+In addition, [petejohanson] did the upgrade work to adjust ZMK for the Zephyr changes.
+
+- Updated to newer devicetree/driver Zephyr API
+- Adjustment for Zephyr pinmux changes
+- Fixes for power management, LVGL, and formatter changes
+
+## Getting The Changes
+
+Use the following steps to update to the latest tooling in order to properly use the new ZMK changes:
+
+### User Config Repositories
```

Comment:
Less technical users might not instantly understand `User Config Repositories` if they're scan-reading the headers only.  Perhaps include the terms `GitHub` and/or `GitHub Actions` in the heading somehow.